packages/venia-ui/lib/components/CheckoutPage/ItemsReview/itemsReview.css

```diff
@@ -1,12 +1,60 @@
-.text_content {
-    padding: 25px;
-    padding-top: 50px;
+.total_quantity {
     font-weight: bold;
 }
 
-.container {
-    min-height: 20rem;
+.show_all_items_button {
+    composes: root_normalPriority from '../../Button/button.css';
```

Comment:
Our normal button styles have very little to do with the button you're styling here. I would suggest not extending our normal button and instead styling this button from scratch.